RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -174,6 +174,13 @@
         'keep floatedmValueMap_lowPtGsfElectronID_*_*')
 )
 
+# Extra modules for bParking era
+from Configuration.Eras.Modifier_bParking_cff import bParking
+_bParking_extraCommands = ['keep recoGsfElectrons_lowPtGsfElectronsPreRegression_*_*']
+bParking.toModify(RecoEgammaFEVT, outputCommands = RecoEgammaFEVT.outputCommands + _bParking_extraCommands)
+bParking.toModify(RecoEgammaRECO, outputCommands = RecoEgammaRECO.outputCommands + _bParking_extraCommands)
+bParking.toModify(RecoEgammaAOD, outputCommands = RecoEgammaAOD.outputCommands + _bParking_extraCommands)
+
```

Comment:
IIUC, the `lowPtGsfElectronsPreRegression` does not exist anymore at runtime and this addition is not really needed

Reply:
True, but I don't see easily how to revert while keeping a sensible config for the LowPtGsfElectronFinalizer.

Reply:
Agree, but I don't see easily how to revert.